Tasks/DownloadPipelineArtifactV2/task.json

```diff
@@ -243,5 +243,11 @@
             "target": "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTaskV2_0_0, Agent.Plugins"
         }
     },
-    "messages": {}
+    "messages": {},
+    "outputVariables": [
+        {
+            "name": "BuildNumber",
```

Comment:
The name is confusing. Should be PipelineId.

Reply:
Naming it "BuildNumber" so it's consistent with build artifacts tasks